src/aioquic/quic/connection.py

```diff
@@ -2317,6 +2346,17 @@ def _write_application(
                         ),
                     )
 
+            # RESET STREAM
+            for stream in self._streams.values():
+                if self._reset_pending:
```

Comment:
A reset is specific to a stream, so the _reset_pending flag should an attribute of the stream, not the connection